providers/kind/kind.sh

```diff
@@ -90,7 +90,7 @@ _setup_kind() {
 	run kind create cluster --name="${KIND_CLUSTER_NAME}" -v6 --config="$KIND_CONFIG_YAML"
 
 	info "Generating kubeconfig"
-	kind get kubeconfig --name="${KIND_CLUSTER_NAME}" | sed 's/kind-kind/kind-kepler/' > "$KIND_KUBECONFIG"
```

Comment:
are we save to remove sed here?

Reply:
Yeah, it just replaces `kind-kind` with `kind-kepler`. I am not sure why we added this in the first place🤔